gutenberg.php

```diff
@@ -110,6 +94,31 @@ function gutenberg_menu() {
 }
 add_action( 'admin_menu', 'gutenberg_menu', 9 );
 
+/**
+ * Site editor's Menu.
+ *
+ * Adds a new wp-admin menu item for the Site editor.
+ *
+ * @since 9.4.0
+ */
+function site_editor_menu() {
+	if ( gutenberg_is_fse_theme() ) {
+		add_menu_page(
+			__( 'Site Editor (beta)', 'gutenberg' ),
+			sprintf(
+			/* translators: %s: "beta" label. */
+				__( 'Site Editor %s', 'gutenberg' ),
+				'<span class="awaiting-mod">' . __( 'beta', 'gutenberg' ) . '</span>'
+			),
+			'edit_theme_options',
+			'gutenberg-edit-site',
+			'gutenberg_edit_site_page',
+			'dashicons-layout'
+		);
+	}
+}
+add_action( 'admin_menu', 'site_editor_menu', 9 );
```

Comment:
Maybe 'gutenberg_site_editor_menu' to be more specific about where its coming from?

Reply:
Good call, updated in 98e635d07eaf06a7d5a89e32c8d97d209517a350.